src/DataTypes/Serializations/ISerialization.h

```diff
@@ -241,6 +241,7 @@ class ISerialization : private boost::noncopyable, public std::enable_shared_fro
     {
         SubstreamPath path;
         bool position_independent_encoding = true;
+        bool enumerate_dynamic_streams = true;
```

Comment:
It would be helpful to add a comment explaining what the setting does.

Reply:
Sure